torch/fx/graph_module.py

```diff
@@ -41,6 +41,7 @@ class GraphModuleImpl(cls):  # type: ignore
     def __init__(self, root: torch.nn.Module, graph: Graph):
         super().__init__()
         self.root = root
+        self.training = self.root.training
```

Comment:
can I make this change here for now to unblock this PR? @jamesr66a @suo

Reply:
Where is this actually used? It shouldn't matter if the top-level module has `training` set; the only thing that this affects is stuff like batch norm. Where are we actually depending on this flag?

Reply:
people might be relying on the training attribute of the graph module, e.g. https://github.com/pytorch/pytorch/blob/master/test/quantization/test_quantize_fx.py#L873